test/remote/gateways/remote_safe_charge_test.rb

```diff
@@ -200,7 +200,7 @@ def test_successful_authorize_and_capture_with_more_options
     auth = @gateway.authorize(@amount, @credit_card, extra)
     assert_success auth
 
-    assert capture = @gateway.capture(@amount, auth.authorization)
+    assert capture = @gateway.capture(@amount, auth.authorization, extra)
     assert_success capture
     assert_equal 'Success', capture.message
```

Comment:
There wasn't a need to write a new remote test, adding this to the existing test should cover this field being passed. For some reason, none of the fields in this `extra` object are being mapped in the response. I paired with Mo on this and we were both stumped by this. We tried to write an integration test in core, but the test file there needs to be updated. Jimil submitted a PR for that.